StackOverflow/Model/QuestionsData.m

```diff
@@ -0,0 +1,13 @@
+//
+//  QuestionsData.m
+//  StackOverflow
+//
+//  Created by Malea on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import "QuestionsData.h"
+
+@implementation QuestionsData
```

Comment:
If this class has no behaviour do we actually need it, or is it a DTO, in which case it should be named as such.